ironfish-cli/src/commands/wallet/send.ts

```diff
@@ -44,6 +44,10 @@ export class Send extends IronfishCommand {
       default: false,
       description: 'Confirm without asking',
     }),
+    confirmations: Flags.integer({
```

Comment:
this `confirmation` flag is very confused with `confirm` flag in the same command. Can you possibly give a different name? eg min blocks